### ISSUE TITLE
Question About Getting Wheels For Library :)

### ISSUE THREAD

**Issue by djmrr** [sethmlarson/hstspreload#6]
Hey there!

Hope all has been well!
I really dig this library that's been crafted!

I was wondering if it would ever be possible to get the binary blobs / wheels for the library:
```
pip3 download --only-binary=:all: hstspreload 
```
or via:
```
pip download --only-binary=:all: hstspreload 
```
To avoid this:
```
ERROR: Could not find a version that satisfies the requirement hstspreload (from versions: none)
ERROR: No matching distribution found for hstspreload
```
Thanks for any input / feedback!

**Comment by sethmlarson** [comment]
This wouldn't be too hard to implement, change the `deploy` task within `noxfile.py` to also build a wheel and everything should work!

**Comment by apollo13** [comment]
@sethmlarson I have added a PR which should fix this issue. I can successfully generate the wheel locally. The only thing I am not 100% sure is if travis would require a `session.install("wheel")` in addition to twine.

**Comment by sethmlarson** [comment]
Closed in #7, thanks all!
